.coveragerc

```diff
@@ -1,4 +1,9 @@
 [run]
 parallel = True
 data_file = /tmp/${USER}/cov/coverage.db
-omit = */site-packages/*
+source =
+    lib/pbench
+omit =
+    */site-packages/*
+    /tmp/${USER}/tox/*/bin/bottle.py
```

Comment:
This seems oddly specific; e.g., note that `gunicorn` and `alembic` which we use on the server side are here, too. Should we be (_can_ we?) ignore everything here except `pbench_*`??

Reply:
This is the only one that showed up in my environment.  I am not sure how to only include `pbench_*` names while excluding others.

Reply:
I've added comments here to see if this helps make things clearer.

Reply:
As @webbnh points out, `source` (unfortunately) causes all other python code to be ignored.  So the `omit` is no longer needed.